scripts/node_e2e_installer

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+NODE_E2E_COMMIT=4d14fd61d2d533a0bb81557b54d3505be830efa7  # Commit to run upstream node e2e tests
+
+enable_selinux () {
+    # Make sure SELinux is enabled
+    /usr/sbin/setenforce 1
+
+    # Get the SELinux package
+    /usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error -o /tmp/kubelet-e2e.pp https://storage.googleapis.com/cri-o/selinux/kubelet-e2e.pp
+    /usr/sbin/semodule -i /tmp/kubelet-e2e.pp
+}
+
+install_crio () {
+    # Download and install CRIO
+    /usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error -o /usr/local/crio-install.sh  https://raw.githubusercontent.com/cri-o/cri-o/master/scripts/get
```

Comment:
Is't `$PATH` set correctly during script execution that we could just
```suggestion
    curl --fail --retry 5 --retry-delay 3 --silent --show-error -o /usr/local/crio-install.sh  https://raw.githubusercontent.com/cri-o/cri-o/master/scripts/get
```

Reply:
Thanks, updated.